homeworks/vladimir.kosenko_soldier3/homework_2/task_2.js

```diff
@@ -0,0 +1,84 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+
+function checkImputValue(min, max) {
+    return ((!Number.isNaN(min) && min > 500 && min < 1000)
+            && (!Number.isNaN(max) && max > 500 && max < 1000));
+}
+
+function getMinMaxArr(minInt, maxInt) {
+    if (minInt > maxInt) {
+        const max = minInt;
+        const min = maxInt;
+        return [min, max];
+    }
+    return [minInt, maxInt];
+}
+
+function getArray(arr) {
+    const nambersArr = [];
```

Comment:
Pls, use english, not anglyiskyi

Reply:
And check js naming conventions.. if you use name in plural form, it implies that it is of array or collection type))